### PR TITLE
Created App navigation routes

### PR DESCRIPTION
**Pull request by aastikn**
Created navigation routes in the app using stack navigation.
image

### CODE REVIEW

File: components/hostjoinSwitch.tsx [hunk 1]
```diff
@@ -47,7 +52,10 @@ const HostJoinSwitch: React.FC<HostJoinSwitchProps> = (props) => {
       >
         <TouchableOpacity
           activeOpacity={1}
-          onPress={() => updatedSwitchData(1)}
+          onPress={() => {
+            updatedSwitchData(1);
+              hostridefunc();
+          }}
```
Comment (RohanKhatua) [inline components/hostjoinSwitch.tsx]: abstract to a different function

**Review by RohanKhatua — CHANGES_REQUESTED** [review]
see review comms

**Review by anandrajaram21 — APPROVED** [review]
lgtm

**Comment by aastikn** [comment]
Resolves #42  : App Routing

**Pull request merged**